### PR TITLE
Add Code tab to Header page

### PR DESCRIPTION
**Pull request by Lisa159**

### CODE REVIEW

File: website/screens/components/header/code/HeaderCodePage.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            <Code>0</Code>
+          </td>
+          <td>
+            Value of the tabindex for all interactuable elements, except those
+            inside the custom area.
+          </td>
+        </tr>
+      </DxcTable>
+    ),
+  },
```
Comment (raquelarrojo) [inline website/screens/components/header/code/HeaderCodePage.tsx:87]: Please add information about `DxcHeader.Dropdown`.

File: website/screens/components/header/code/HeaderCodePage.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    subSections: [
+      {
+        title: "Basic usage",
+        content: (
+          <>
+            <Example example={basicUsage} defaultIsVisible />
+          </>
+        ),
+      },
+      {
```
Comment (raquelarrojo) [inline website/screens/components/header/code/HeaderCodePage.tsx:99]: Please remove React.Fragments in all the examples `<></>`.

File: website/screens/components/header/usage/HeaderUsagePage.tsx [hunk 3]
```diff
@@ -4,8 +4,9 @@ import Figure from "@/common/Figure";
 import Image from "@/common/Image";
 import QuickNavContainer from "@/common/QuickNavContainer";
 import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
-import headerVariants from "./images/header_variants.png";
 import headerResponsive from "./images/header_responsive.png";
```
Comment (raquelarrojo) [inline website/screens/components/header/usage/HeaderUsagePage.tsx:7]: Please remove also the `header_variants` image from `/images`.

**Review by raquelarrojo — CHANGES_REQUESTED** [review]

File: website/screens/components/header/code/HeaderCodePage.tsx (hunk 1 above)
Comment (Lisa159) [reply website/screens/components/header/code/HeaderCodePage.tsx:94]: Done

File: website/screens/components/header/code/HeaderCodePage.tsx (hunk 2 above)
Comment (Lisa159) [reply website/screens/components/header/code/HeaderCodePage.tsx:122]: Done

File: website/screens/components/header/usage/HeaderUsagePage.tsx (hunk 3 above)
Comment (Lisa159) [reply website/screens/components/header/usage/HeaderUsagePage.tsx:7]: Done

**Review by raquelarrojo — APPROVED** [review]

**Pull request merged**